src/Nexosis/Api/Data.elm

```diff
@@ -49,12 +49,12 @@ get config page pageSize sorting =
             pageParams page pageSize
                 ++ sortParams sorting
     in
-    (getBaseUrl config ++ "/data")
-        |> HttpBuilder.get
-        |> HttpBuilder.withExpectJson decodeDataSetList
-        |> HttpBuilder.withQueryParams params
-        |> addHeaders config
-        |> HttpBuilder.toRequest
+        (getBaseUrl config ++ "/data")
```

Comment:
A bunch of these indentation changes are just from saving the file. So I couldn't undo them because they are redone every time I save.